src/main/java/io/kestra/plugin/docker/Compose.java

```diff
@@ -54,14 +89,24 @@
     }
 )
 public class Compose extends AbstractExecScript implements RunnableTask<ScriptOutput> {
+    private static final String DEFAULT_IMAGE = "docker:latest";
 
     @Schema(
         title = "The compose file (can be passed as an inline YAML, a path in working directory, or a Kestra URI)"
     )
-    @NotNull
     @PluginProperty(internalStorageURI = true)
     private Property<String> composeFile;
 
+    @Schema(
+        title = "List of Compose files to load.",
+        description = "If multiple files are provided, they are passed in order using repeated `-f` flags."
+    )
+    @PluginProperty(internalStorageURI = true)
+    private Property<List<String>> composeFiles;
```

Comment:
Should it be a `Object from` to be able to take paths from the internal storage?